plugins/env/app/views/samson_env/_deploy_tab_nav.html.erb

```diff
@@ -1 +1,23 @@
-<li><a href="#environment-variables" data-toggle="tab">Environment variables</a></li>
+<%
+  previous_env = @deploy.stage ? @deploy.previous_successful_deploy&.env_state : nil
+  current_env = @deploy.project ? @deploy.retrieve_env_state : nil
+  diff = (previous_env.presence != current_env.presence)
```

Comment:
Might be a little more clean to have this as a cached value in a helper method rather than using `content_for` between views?

Reply:
yeah was my first thought too, but then it's a shared instance var that gets cached ... which is kinda funky/complex too ... I'll check out how it looks and merge if it's uglier ...

Reply:
yeah it's ugly ... need to cache false then and assign arrays and stuff ...

Reply:
but testing breaks ... so I guess helper it is ...